model/student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯\\(ツ)/¯'
   end
+
+  def classroom(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.animals.includes?(self)
```

Comment:
- [ ] I think you want to check for a condition that verifies if the `classroom.students` already have the student, I observed there is a little mistake not **classroom.animals.includes?(self)**.
```suggestion
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
```

- [ ] Please don't forget to indicate classroom as `attr_reader` as well.

Reply:
thank you @OyePriscilla , i will make changes and request a new review